stylus/ui-components/button.styl

```diff
@@ -106,6 +106,11 @@ $button--upload
     background-position  em(17px, 16px) center
     background-repeat    no-repeat
 
+$button--upload-menu-item
```

Comment:
The point is precisely that only upload button in a menu should have those properties.

To mutualize this ruleset, another way should be to declare `coz-btn--upload-primary` for the actual `coz-btn--upload` and use `coz-btn--upload` for this one.

I also hesitated to not use a `coz-btn-*` class but some kind of `coz-menu-item--upload`. Maybe it's the right solution as in the case of Cozy-Photos-v3 this element does not have other button classes like `coz-btn`, `coz-btn--upload`.